front/src/common/Map/Search/useSearchOperationalPoint.tsx

```diff
@@ -112,7 +112,7 @@ export default function useSearchOperationalPoint({
 
         if (chCodeFilter === undefined) return true;
 
-        return result.ch.toLocaleLowerCase().includes(chCodeFilter.trim().toLowerCase());
+        return result?.ch?.toLocaleLowerCase().includes(chCodeFilter.trim().toLowerCase());
```

Comment:
I don't know if it's necessary, but if it makes the code more robust we should keep it

Reply:
i encountered some issues not relating to this where i have a blank page :/

Reply:
Hum are you sure it is necessary ? Can you check again now ? :/

Reply:
Done !